aten/src/ATen/native/quantized/cpu/qconv.cpp

```diff
@@ -310,6 +310,53 @@ class QConv2dInt8 final : public c10::OperatorKernel {
         output_min,
         output_max);
 
+    auto input_scale = input_contig.q_scale();
+
+    // Re-quantizing the bias based on input scale and weight scale.
+    if (input_scale != pack_ptr.input_scale) {
```

Comment:
you probably need to check zero point too

Reply:
Input zero point is not used in the pre-packing step anymore, so its fine if it changes at runtime.

Reply:
For bias, the quantization depends only on the input scale, not the zero-point